### PR TITLE
Removes corresponding SCSS and JSX for archived pages top bar

### LINKED ISSUE #3538: [Minor] Issue with archived pages, top bar is missing z-index
1. Set the screen resolution to 1920x1080
2. Go to https://developer.mozilla.org/en-US/docs/Mozilla/Performance/Scroll-linked_effects (or any other page using this banner)
3. Observe the archived page bar

**Expected:**

I expected the top bar to be in the middle

**Actual:**

The text in the top bar is shifted to the right

image

**Fix**

Add `left: 0;` to `.archived {` in the css file `index.scss`

### PR DESCRIPTION
**Pull request by tannerdolby**
Fixes #3538 

@peterbe We can let this one sit like you mentioned until after the first week of July when the quarter ends.

### CODE REVIEW

**Comment by tannerdolby** [comment]
Woops, forgot to actually remove the `Archived` component where it was being used further down. Hence the build failing at Lint ESlint.

```
/home/runner/work/yari/yari/client/src/document/index.tsx
Error:   153:47  error  'Archived' is not defined  react/jsx-no-undef
```

tests should pass now :)

**Review by peterbe — APPROVED** [review]
Excellent. But to be perfectly safe, just in case we do need to run another prod build of the archived pages, let's not merge this until after we've pruned the archived content from prod. 

@schalkneethling what do you think?

CC @escattone

**Comment by peterbe** [comment]
@tannerdolby We are rolling out the deletion of archived content in the next build, tonight. Once that's been done, this PR of yours should be ready to land. Would you mind doing one last conflict resolution so I can merge it tomorrow.

**Comment by tannerdolby** [comment]
> We are rolling out the deletion of archived content in the next build, tonight. Once that's been done, this PR of yours should be ready to land. Would you mind doing one last conflict resolution so I can merge it tomorrow.

@peterbe Ok cool, thanks for the heads up. Yeah sure thing, I will resolve the conflicting files shortly.

File: client/src/document/index.tsx
Comment (peterbe) [inline client/src/document/index.tsx:153]: You can now delete this line.

Comment (tannerdolby) [reply client/src/document/index.tsx:153]: Woops! Almost forgot about this, good catch.

**Comment by tannerdolby** [comment]
I reckon this should be good to go now @peterbe !

**Pull request merged**
